sdks/jwt/jwt/signer/c12d7914-4e38-4b42-a221-2c27e7aa6bda.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "sdks/jwt/JWT/Signer",
```

Comment:
I think it's unnecessary to add an extra level in the namespace. `sdks/jsonwebtoken/Signer` should work? Same for `Verifier`